variables.tf

```diff
@@ -75,3 +75,12 @@ variable "ec2_ingress_sg_id" {
   type        = list(string)
   default     = []
 }
+
+//----------------------------------------------------------------------
+// Networking Variables
+//----------------------------------------------------------------------
+variable "container_insights_enablement" {
+  description = "Whether contain sights are set  Value values are [enabled',disabled]"
```

Comment:
Could be a couple of typos here (double-space before "Value", should "Value" read "Valid", an odd single quote after enabled).

Reply:
Updated